src/atoms/Navbar/Navbar.js

```diff
@@ -243,6 +243,19 @@ class Navbar extends Component {
 		}));
 	};
 
+	checkListComponent = () => {
+		const { children } = this.props;
+		let isList = false;
+		React.Children.forEach(children, child => {
```

Comment:
I think it's better to use one of `find`, `every` or `some` instead of forEach.

Reply:
We only have few methods on `React.Children` & I cool find only this helpful. Check this: https://reactjs.org/docs/react-api.html#reactchildren .

We can try with `.toArray` but not sure how it would fit in our case.

Reply:
https://github.com/appbaseio/reactivesearch/blob/next/packages/web/src/components/result/ResultList.js#L22

Reply:
Cool got it. Thanks 💯